observability/logging/logging-6.2/log6x-configuring-lokistack-otlp-6.2.adoc

```diff
@@ -0,0 +1,145 @@
+:_mod-docs-content-type: ASSEMBLY
```

Comment:
I think that all of the headings in this Assembly need to be modularized into either CONCEPT or PROCEDURE. I'll comment these headings, but please reach out if you have questions. 

https://redhat-documentation.github.io/modular-docs/#assembly-definition